ols/app/models/models.py

```diff
@@ -220,6 +222,13 @@ class FeedbackRequest(BaseModel):
         }
     }
 
+    def __init__(self, **data: Any) -> None:
+        """Initialize configuration and perform checking of provided attributes."""
+        super().__init__(**data)
+        # check if conversation ID has the proper format
+        if not (suid.check_suid(self.conversation_id)):
+            raise ValueError(f"Improper conversation ID {self.conversation_id}")
```

Comment:
Why not use [field validator](https://docs.pydantic.dev/latest/concepts/validators/#validation-of-default-values) for this?

Reply:
we can

Reply:
updated @onmete